perl.md

```diff
@@ -0,0 +1,52 @@
+date = "2022-01-12T00:23:27Z"
+title = "Perl in WebAssembly"
+description = "A Perl interpreter in WebAssembly is available."
+tags = ["perl", "language", "webassembly"]
+template = "page_lang"
+[extra]
+author = "Fermyon Staff"
+---
+# Perl in WebAssembly
+
+Perl (Practical Extraction and Reporting Language) was the _lingua franca_ of the early web.
+A powerful scripting language, it became a popular way to write CGI.
+
+Larry Wall, its creator, once called it the "first post-modern programming language"
+because it was driven more by a desire to add the things developers liked than to follow specific academic strictures.
+Its regular expressions have lived on, influencing the designs of countless
+other programming languages.
+
+## Available Implementations
+
+There is no official Perl distribution that is WebAssembly-capable.
+
+The [WebPerl project](https://webperl.zero-g.net/) provides a browser-centered Perl interpreter in Wasm.
+Build instructions support Perl 5.26 and newer.
+
+WebPerl does not provide a WASI implementation, and does not appear to be runnable outside of a browser.
+
+## Usage
+
+WebPerl describes [usage inside of a browser](https://webperl.zero-g.net/using.html).
+
+## Pros and Cons
+
+Things we like:
+
+- The project supports a tremendous number of features, including a way to bundle PMs.
```

Comment:
What are PMs?  If the acronym is common-knowledge for Perl users, no need to spell out... but if there's a chance it isn't widely known, perhaps we should.

Reply:
Perl modules. It was common knowledge the last time I wrote perl....10 years ago...